code/modules/mob/living/simple_animal/hostile/pitbull.dm

```diff
@@ -91,3 +91,41 @@
 				qdel(src)
 		else
 			gib()
+
+/mob/living/simple_animal/hostile/pitbull/smashednslammed
+	name = "toadline pitbull"
+	icon_state = "toadlinepitbull"
+	icon_living = "toadlinepitbull"
+	icon_dead = "toadlinepitbull_dead"
+	pass_flags = PASSMOB|PASSDOOR //so smashed and slammed they can squeeze through doors
+	health = 35
+	maxHealth = 35
+	speed = 3 //stumpier legs move slower
+
+	melee_damage_lower = 4
+	melee_damage_upper = 6 //stumpy but they hit harder
+
+/mob/living/simple_animal/hostile/pitbull/smashednslammed/New()
+	..()
+	desc = pick(
+		"SMASHED and SLAMMED",
+		"Certified Toadline.",
+		"Help I can't breathe",
+		"Bloodskull!",
+		"Two times Pimpy...",
+		"Three times Bape!",
+		"Not a family tree, but a family circle.")
+
+/mob/living/simple_animal/hostile/pitbull/smashednslammed/summoned_pitbull
+	faction = "wizard" // so they get along with other wizard mobs
+	meat_type = /obj/item/weapon/ectoplasm //a magical dog
+	
+/mob/living/simple_animal/hostile/pitbull/smashednslammed/summoned_pitbull/death(var/gibbed = FALSE)
+	..()
+	if(!gibbed)
+		if(prob(90))
+			animate(src, alpha = 0, time = 4 SECONDS)
+			spawn(4 SECONDS)
+				qdel(src)
+		else
+			gib()
```

Comment:
This is unnecessary as this behavior is already inherited from the parent proc.

Reply:
Smashed and slammed is not a child of the summoned pitbulls (which would make that code redundant), it's a child of regular pitbulls, meaning the summoned one is a child of the toadline pitbulls (which in turn is a child of regular pitbulls), it's not redundant code.
As for why have regular non summonable smashed pitbulls, there are some non wizard sourced, like the syndie pokeballs or the goldcore reactions.

Reply:
Christ, that's a mess. I won't force you to, but cleaning up code here for consistency would be nice.